tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -370,7 +368,7 @@ private async Task AddMicrosoftWebApp_TestRedirectToIdentityProviderEvent(IServi
             await oidcOptions.Events.RedirectToIdentityProvider(redirectContext).ConfigureAwait(false);
 
             // Assert properties set, events called
-            await redirectFunc.ReceivedWithAnyArgs().Invoke(Arg.Any<RedirectContext>()).ConfigureAwait(false);
+          //  await redirectFunc.ReceivedWithAnyArgs().Invoke(Arg.Any<RedirectContext>()).ConfigureAwait(false);
```

Comment:
@pmaytak : do we need this line now?
If yes, how to fix it?

Reply:
Leave it commented for this PR and I will investigate and propose a different PR.

I would think it should work since we should be able to still resolve OIDC options. I will take a look afterwards.

Reply:
@jmprieur I sent a PR #291 to fix these tests. 
These tests and similar ones test that we call the custom event handlers set up by the dev.

Reply:
Thanks @pmaytak